packages/twenty-server/src/database/clickhouse/migrations/001-create-events-table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS events
+(
+    `action`      LowCardinality(String),
+    `timestamp`   DateTime64(3),
+    `version`     LowCardinality(String),
+    `userId`      String DEFAULT '',
+    `workspaceId` String DEFAULT '',
+    `payload`     String
+)
+    ENGINE = MergeTree
```

Comment:
I read about partition and I think we can remove the partition key entirely 
See for example https://stackoverflow.com/a/79372034

Reply:
Fix in https://github.com/twentyhq/twenty/commit/100ee351191f41a770537aad0e2e2a25edb3215d